ietf-optical-impairment-topology.tree

```diff
@@ -95,11 +92,11 @@ module: ietf-optical-impairment-topology
                    |     |       frequency-thz
                    |     +--ro minimum-channel-spacing?
                    |     |       frequency-ghz
-                   |     +--ro min-channel-output-power?   dbm-t
-                   |     +--ro max-channel-output-power?   dbm-t
-                   |     +--ro min-channel-input-power?    dbm-t
-                   |     +--ro max-channel-input-power?    dbm-t
-                   |     +--ro max-total-input-power?      dbm-t
+                   |     +--ro tx-channel-power-min?      dbm-t
+                   |     +--ro tx-channel-power-max?      dbm-t
+                   |     +--ro rx-channel-power-min?      dbm-t
+                   |     +--ro rx-channel-power-max?      dbm-t
+                   |     +--ro rx-total-power-max?        dbm-t
```

Comment:
power min/max and minimum-channel-spacing are already defined in the Application Code of G.698.2 (see Table 8-7) and would need to go to the explicit mode

Reply:
This is in fact added for "organizational" mode, that it is not mandating that follows the G.692 so these parameters needs to be considered. This is the reason for which a common grouping between organizational and explicit have been defined, and not for application code.